Makefile

```diff
@@ -26,6 +26,12 @@ test: yarn.lock
 		--workdir /usr/src/app \
 		$(IMAGE_NAME) npm run $(NPM_TEST_TARGET)
 
+yarn.add:
+	docker run -ti --rm \
+		-v $(PWD):/usr/src/app \
```

Comment:
We have a shell style guide somewhere that encourages us to write the long form of flags when possible to enhance readability, (tty, interactive, volume, I think, in this case).

Reply:
I think we have a yarn task which could be used to accomplish this goal. I don’t mind adding another with better ergonomics, but should we document it in the contributors file?

Reply:
yeah, we have one to generate the lock, but meaning we should change `package.json` manually, this would accomplish both things.

Reply:
> We have a shell style guide somewhere that encourages us to write the long form of flags when possible to enhance readability, (tty, interactive, volume, I think, in this case).

I will keep it like this to be consistent with the existing targets and assuming those are well known flags (`-ti --rm -v`)